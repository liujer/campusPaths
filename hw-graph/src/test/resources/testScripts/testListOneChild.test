# Tests the method listChildren of DLGraph by adding
# multiple nodes with the same children
CreateGraph B
AddNode B one
AddNode B two
AddNode B three
AddEdge B one two e12
AddEdge B three two e32
ListChildren B one
ListChildren B two
ListChildren B three