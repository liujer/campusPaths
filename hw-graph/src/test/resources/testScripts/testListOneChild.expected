# Tests the method listChildren of DLGraph by adding
# multiple nodes with the same children
created graph B
added node one to B
added node two to B
added node three to B
added edge e12 from one to two in B
added edge e32 from three to two in B
the children of one in B are: two(e12)
the children of two in B are:
the children of three in B are: two(e32)